tests/test_config_reader.py

```diff
@@ -580,3 +582,82 @@ def test_resolve_node_tag(self):
         new_node = config_reader.resolve_node_tag(mock_loader, mock_node)
 
         assert new_node.tag == "new_tag"
+
+    def test_render_missing_config_file(self):
```

Comment:
I've found it's more helpful in test names if you follow the format of `test_{method under test}__{situation under test}__{expected behavior or result}`. 

So, for example, in this case: `test_render__missing_config_file__returns_none`. That way when you read the test name, you know what the test is about. And when the test fails, it's absolutely clear what case and result _failed_.

Reply:
ok done